components/list/list-item.js

```diff
@@ -72,6 +70,17 @@ class ListItem extends ListItemCheckboxMixin(ListItemRoleMixin(RtlMixin(LitEleme
 			:host(:last-child) d2l-list-item-generic-layout[data-separators="between"] {
 				border-bottom: 1px solid transparent;
 			}
+			:host([dragging]) d2l-list-item-generic-layout {
```

Comment:
Oh hmmm... I guess maybe this does need to remain public.

Reply:
It is read only. Give the ability to style the left over image when it is being dragged.